account_payment_order_notification/wizard/wizard_account_payment_order_notification.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.tests.common import Form
+
+
+class WizardAccountPaymentOrderNotification(models.TransientModel):
+    _name = "wizard.account.payment.order.notification"
+    _description = "Wizard Account Payment Order Notification"
+
+    payment_order_id = fields.Many2one(
+        comodel_name="account.payment.order", required=True, readonly=True,
+    )
+    mail_template_id = fields.Many2one(
+        comodel_name="mail.template",
+        domain=[
+            ("model_id.model", "=", "account.payment.order"),
+            ("auto_delete", "=", True),
```

Comment:
Why auto-delete? And this arises another question? What happen with the replies? They shouldn't go to the payment order, but they shouldn't be lost as well. Maybe we need another model "account.payment.order.notification" for hosting them, with the partner, related lines, and putting as follower the followers of the payment order.